zerver/openapi/zulip.yaml

```diff
@@ -20550,6 +20563,10 @@ paths:
                               type: array
                               description: |
                                 The integer user IDs of the user group members.
```

Comment:
I am not sure whether we want to mention here that only active users are included. Like we do not mention this for stream subscribers and it is kind of understood that only active users should be included.

Reply:
I'll think on this. Probably just going to do a bonus copyediting commit.